client/src/component/FAQ/FAQ.jsx

```diff
@@ -0,0 +1,123 @@
+
+
+import React from "react";
+import 'font-awesome/css/font-awesome.min.css'
+import './FAQ.css'
+
+
+
+const Faqs = () => (
+  <div className="Faqs" id="Faqs">
+      <h2>
+        <u>FREQUENTLY ASKED QUESTIONS</u>
+      </h2>
+      <div className="container">
+        <div className="column left">
```

Comment:
having it as `        <div className="column left">` invokes both the `column` CSS and `left` css.
```suggestion
        <div className="column-left">
```
This would give you the option to add custom column CSS to your file.
or you could make it `<div className="column">` to keep the column width.